cypress/e2e/smoke/pending_actions.cy.js

```diff
@@ -0,0 +1,68 @@
+const SAFE = 'gor:0xCD4FddB8FfA90012DFE11eD4bf258861204FeEAE'
+
+describe('Pending actions', () => {
+  before(() => {
+    cy.connectE2EWallet()
+
+    cy.visit('/welcome')
+
+    // Close cookie banner
+    cy.contains('button', 'Accept all').click()
+
+    // Ensure wallet is connected to correct chain via header
+    cy.contains('E2E Wallet @ Görli')
+  })
+
+  beforeEach(() => {
+    cy.restoreLocalStorageCache()
+  })
+
+  afterEach(() => {
+    cy.saveLocalStorageCache()
+  })
```

Comment:
Can you add a comment why this is necessary?

Reply:
WDYT?
```suggestion
  beforeEach(() => {
    // Uses the previously saved local storage
    // to preserve the wallet connection between tests
    cy.restoreLocalStorageCache()
  })

  afterEach(() => {
    cy.saveLocalStorageCache()
  })
```

Reply:
Looks good to me.